src/memcached_cache.cpp

```diff
@@ -615,13 +615,43 @@ Store::Status MemcachedCache::update_irs_associated_impus(MemcachedImplicitRegis
   return status;
 }
 
+// Create an IMPU from an IRS and set it in the store.
+// This is used when we think that the IMPU is new.
```

Comment:
I'd clarify this and say `This is used when we failed to find the IMPU in the store, and have thus created a new record for it`.